services/QuillLMS/spec/models/user_spec.rb

```diff
@@ -901,6 +901,11 @@
         expect(user).to be_valid
       end
 
+      it 'is invalid when there is a space in it' do
+        user = build(:user,  email: 'test@test.lan ')
+        expect(user).to_not be_valid
```

Comment:
It may also be worth confirming that `user.errors[:email]` includes our specific error for this class just in case sometime in the future we do something crazy like make `.lan` email addresses invalid and then break the spacing invalidation.  (Very unlikely, but maybe worth covering anyway.)